.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDatePeriodParser.cs

```diff
@@ -1024,7 +1099,7 @@ private DateTimeResolutionResult ParseOneWordPeriod(string text, DateObject refe
                     if (this.config.IsWeekOnly(trimmedText))
                     {
                         var monday = referenceDate.This(DayOfWeek.Monday).AddDays(7 * swift);
-                        ret.Timex = DateTimeFormatUtil.ToIsoWeekTimex(monday);
+                        ret.Timex = isReferenceDatePeriod ? TimexUtility.GenerateWeekTimex() : DateTimeFormatUtil.ToIsoWeekTimex(monday);
```

Comment:
Also a method in timexutil.

Reply:
replaced DateTimeFormatUtil.ToIsoWeekTimex with TimexUtility.GenerateWeekTimex